playhouse/tests/test_keys.py

```diff
@@ -334,17 +334,28 @@ def setUp(self):
         Language.drop_table(True)
         Language.create_table()
         Snippet.create_table()
+        Snippet2.drop_table(True)
+        Language2.drop_table(True)
+        Language2.create_table()
+        Snippet2.create_table()
 
     def tearDown(self):
         super(TestDeferredForeignKey, self).tearDown()
         Snippet.drop_table(True)
         Language.drop_table(True)
+        Snippet2.drop_table(True)
+        Language2.drop_table(True)
 
     def test_field_definitions(self):
         self.assertEqual(Snippet._meta.fields['language'].rel_model, Language)
         self.assertEqual(Language._meta.fields['selected_snippet'].rel_model,
                          Snippet)
 
+    def test_field_definitions2(self):
+        self.assertEqual(Snippet2._meta.fields['language'].rel_model, Language2)
+        self.assertEqual(Language2._meta.fields['selected_snippet'].rel_model,
+                         Snippet2)
```

Comment:
Since the test-case is only testing the internal APIs, I think it'd be acceptable to move the model definitions into the test method.

I'll merge and then update the code.,

Thanks!